docs/elements.md

```diff
@@ -1442,6 +1443,38 @@ The HTML inside the inner `pl-variable` tag is used as the description. If the t
 - [demo/autograder/python/plots]
 - [demo/autograder/python/random]
 
+## `pl-sketch` element
+
+Freely draw a sketch of whatever you want.
+
+#### Sample Element
+
+<img src="elements/pl-sketch.png" width="300" />
+<br />
+
+**question.html**
+
+```html
+<pl-sketch width="500" height="600"></pl-sketch>
+```
+
+#### Customizations
+
+| Attribute | Type    | Default | Description                  |
+| --------- | ------- | ------- | ---------------------------- |
+| `width`   | integer | 500     | Width of sketchpad element.  |
+| `height`  | integer | 600     | Height of sketchpad element. |
+
+#### Details
+
+Any submission will be marked correct, as there is no way to automatically grade a sketch.
+
```

Comment:
Do you actually want to grade it? I don't think it should be (see elements like `pl-rich-text-editor` which are also used for input but don't directly affect grading). Even if you do, note that you should consider questions that contain other elements as well as a sketch, and in that case the score of other elements should also be considered. In any case, there is no grade function in your element.